CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+In this repository the `main` branch contains the latest development version of the Ably SDK. All development (bug fixing, feature implementation, etc.) is done against the `main` branch, which you should branch from whenever you'd like to make modifications. Here's the steps to follow when contributing to this repository.
```

Comment:
The headers aren't quite right in this document - needs an `h1` and the current `h3`s should be `h2`s.

Reply:
c86eb036